docs/user-guide.rst

```diff
@@ -207,8 +207,8 @@ The backends available are:
   You need to provide **Sourceforge name** if the name on RSS feed is different then the
   project name on Sourceforge.
 
-* **Sourceforge (git) for projects hosted on
-  `sourceforge.net <https://sourceforge.net>`_
+* **Sourceforge (git)** for projects hosted on
```

Comment:
I'm surprised this wasn't caught by the tests. It was recently added in https://github.com/fedora-infra/anitya/pull/1134.

Reply:
I was too. I looked at it again, Zuul configuration is wrong,
job `anitya-tox-docs` is actually running Bandit.
Probably just a copy paste error and easy to fix?

Otherwise, Vagrant uses the Makefile in the `docs` folder
while Zuul runs some commands embedded inside `tox.ini`.
That could cause varying results, too.

```
- job:
    name: anitya-tox-docs
    ...
    parent: tox
    run: ci/test-tox.yaml
    vars:
      tox_env: bandit
    ...
```

Reply:
This really looks like a copy paste error on my side. I will fix it. I will also unify how the docs are built in vagrant and in tox.

Reply:
I looked at the makefile and the docs tox target and they are almost identical, the tox one is even running `./generate_db_schema`. So I would just fix the zuul file.